### DISCUSSION TITLE
How to Suggest New Game Modes

### DISCUSSION THREAD

**Discussion by chris-bsnake** [BattlesnakeOfficial/feedback#31] [Announcements & News]
We love it when community members come up with new ideas for rules and game modes that change how Battlesnake works and that create new puzzles for the community to solve. If you have an idea for a new game mode, just create a new discussion under the 'Game Mode' category in this project. Admins will review the suggestions and let you know if there are enough details and provide some high level feedback.  Then the community can vote on the new mode to determine which ones may end up in the Battlesnake Roadmap.

### Submission Guidelines

1. Set the title of the issue to `New Game Mode: _________` with the blank replaced by the name for your game mode
2. Provide a TLDR; of the concept of your idea
3. Provide a description of how the game mode changes the core rules of Battlesnake:
- Movement
- Board layout
- Food
- Health
4. Add mockups or screenshots to show how the game mode might work 

### Community Voting

Once posted, if you like a new game mode idea, feel free to upvote it and add reactions to show you are in favour of it.  If a new mode gets enough traction the Battlesnake team will consider adding it to the roadmap.

If you have any questions about how to create a submission, feel to reach out to the Battlesnake team on [Discord](https://play.battlesnake.com/discord).

**Comment by bvanvugt** [comment]
(updated to reflect new discussion board setup)
